src/lib/falcor/FalcorController.js

```diff
@@ -74,6 +74,7 @@ export default class FalcorController extends BaseComponent {
 
   // Makes falcor fetch its paths
   falcorFetch(falcorPathSets) {
+    console.log("FATCHANG");
```

Comment:
Yep, for sure going to, as I said in the slack, it's not quite ready yet. It's still good getting your comments now as most stuff is bascially done. But the reason for the pull request was mostly so Zane could see the new falcor path syntax